src/main/java/net/mcreator/ui/modgui/AchievementGUI.java

```diff
@@ -199,7 +199,7 @@ public AchievementGUI(MCreator mcreator, ModElement modElement, boolean editingM
 
 		propertiesPanel.setBorder(BorderFactory.createTitledBorder(
 				BorderFactory.createLineBorder(Theme.current().getForegroundColor(), 1),
-				L10N.t("elementgui.advancement.display_paramters"), 0, 0, propertiesPanel.getFont().deriveFont(12.0f),
+				L10N.t("elementgui.advancement.display_parameters"), 0, 0, propertiesPanel.getFont().deriveFont(12.0f),
```

Comment:
Klemen should confirm this, but generally, we do this type of fix in separate PRs.

Reply:
Yep, we fix those separately. We also generally avoid changing translation key names, even if they have typos because such change reverts translations to other languages at https://crowdin.com/project/mcreator

Reply:
I apologize, should i remove this change from the PR?

Reply:
You should indeed

Reply:
I removed it!